datasets/mur.yaml

```diff
@@ -21,7 +21,7 @@ Tags:
   - weather
 License: There are no restrictions on the use of these data.
 Resources:
-  - Description: MUR Level 4 SST dataset in [Zarr](https://zarr.readthedocs.io/en/stable/) format.
+  - Description: MUR Level 4 SST dataset in [Zarr](https://zarr.readthedocs.io/en/stable/) format. The bucket includes 2 top level directories which contain mostly equivalent zarr stores - zarr/ and zarr-v1/. The only difference between these 2 stores is the chunk shape. The zarr/ directory contains a zarr store chunked (6443, 100, 100) along the dimensions (time, lat, lon) respectively, whereas zarr-v1/ is chunked along the same dimensions but sized (5, 1799, 3600).
```

Comment:
Lets list these as separate resource entries with the prefix, similar to how it is done at https://github.com/awslabs/open-data-registry/blob/master/datasets/nrel-pds-nsrdb.yaml#L25

Reply:
Good idea Zac! Updated to have multiple entries